osmaxx-py/osmaxx/excerptexport/models/bounding_geometry.py

```diff
@@ -10,13 +13,23 @@ def type(self):
         return type(self).__name__
 
     @property
-    def geometry_instance(self):
-        if hasattr(self, 'bboxboundinggeometry'):
-            return self.bboxboundinggeometry
-        elif hasattr(self, 'osmosispolygonfilterboundinggeometry'):
-            return self.osmosispolygonfilterboundinggeometry
-        else:
-            return self
+    def geometry(self):
+        raise NotImplementedError
```

Comment:
Would it make sense to return `self.subclass_instance.geometry` here? This would simplify the implementation of `type_of_geometry`.